### PR TITLE
Sharetribe branding component

### PR DESCRIPTION
**Pull request by mporkola**
React-based branding component for non-whitelabel marketplaces

### CODE REVIEW

**Comment by Gnito** [comment]
Do you have link to styleguide? It could be added here and shared with Juho & Janne too.

File: client/app/components/composites/Branding/Branding.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { t } from '../../../utils/i18n';
+
+import css from './Branding.css';
+
+export default function Branding({ linkToSharetribe }) {
+  const learnMoreLink = a({ href: linkToSharetribe }, t('web.branding.learn_more'));
+  const sharetribeLink = a({ href: linkToSharetribe }, 'Sharetribe');
+
+  return div({
+    className: css.brandingContainer,
```
Comment (Gnito) [inline client/app/components/composites/Branding/Branding.js]: By not giving classname handles, it makes harder for endusers to style or hide this stuff - which is probably OK?

File: client/app/components/composites/Branding/Branding.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.brandingContainer {
+  border-top: 1px solid #EBEBEB;
```
Comment (Gnito) [inline client/app/components/composites/Branding/Branding.css]: Also: 
app/components/composites/Branding/Branding.css
 2:25  ✖  Expected "#EBEBEB" to be "#ebebeb"   color-hex-case

File: client/app/components/composites/Branding/Branding.js (hunk 1 above)
Comment (mporkola) [inline client/app/components/composites/Branding/Branding.js]: Yeah, didn't think custom styling this is something we want :)

**Pull request merged**